docs/source/advanced/training_tricks.rst

```diff
@@ -5,16 +5,21 @@
 
 .. _training_tricks:
 
+###############
 Training Tricks
-================
-Lightning implements various tricks to help during training
+###############
+
+Lightning implements various tricks to help during training:
 
 ----------
 
-Accumulate gradients
---------------------
-Accumulated gradients runs K small batches of size N before doing a backwards pass.
-The effect is a large effective batch size of size KxN.
+********************
+Accumulate Gradients
```

Comment:
should this be part of the optimization section?

Reply:
100% will put it there and render it here as a direct copy.